datamodules/lucas.py

```diff
@@ -36,7 +37,7 @@ class LucasDataModule(pl.LightningDataModule):
     DataModule for Lucas' fusion dataset.
     """
 
-    DATA_FILENAME = "lucas_data_f32.pickle"
+    DATA_FILENAME = "/Users/cusg/Documents/MIT PSFC/projects/Disruption Prediction/hdl_improvement/Model_training/Full_HDL_dataset_unnormalized_no_nan_column_names_w_shot_and_time.pickle"
```

Comment:
Did you make modifications to the dataset? If so I can put them in cloudflare and set up the downloading in the LucasDataLoader

Reply:
oooo no I don't think I changed any numbers specifically so we're OK